client/modules/Slack/SlackBotSender.php

```diff
@@ -41,8 +41,8 @@ class SlackBotSender
      */
     public function __construct($slackBotUrl, $auth = null)
     {
-        $this->slackBotUrl = $slackBotUrl;
-        $this->auth        = $auth;
+        self::$slackBotUrl = $slackBotUrl;
```

Comment:
1. in the class was static not properties - that is wrong
2. static properties should be deleted
3. so need to fix that code in another way, i think

now in the application can not be more than one client.